cms/src/components/content/EditContentForm.vue

```diff
@@ -350,6 +389,76 @@ const checkIfDirty = () => {
                     >
                         Only used for display, does not automatically publish at this date
                     </LInput>
+                    <LInput
```

Comment:
Clearing the input using the browser "clear" button leads to an empty error in the interface:

<img width="740" alt="image" src="https://github.com/bccsa/luminary/assets/39056299/fbef6276-a87d-48a0-be04-8aac19f37ebb">

I ran into the same problem with the publish date, with the problem being the browser. To fix it, copy the `transform` method in the `yup` declaration:
```ts
.transform((value, _, context) => {
                // Check to see if the previous transform already parsed the date
                if (context.isType(value)) {
                    return value;
                }

                // Default validation failed, clear the field
                // This happens when the 'clear' button in the browser datepicker is used,
                // which sets the value to an empty string
                return undefined;
            })
```